src/apis/index.js

```diff
@@ -200,6 +202,8 @@ export default store => {
     setSignatureFonts: setSignatureFonts(store),
     setSelectedTab: setSelectedTab(store),
     getSelectedThumbnailPageNumbers: getSelectedThumbnailPageNumbers(store),
+    selectThumbnailPageNumbers: selectThumbnailPageNumbers(store),
```

Comment:
Personally I feel like `selectThumbnailPages` is better than what we have now. Ending the name with `Numbers` is a bit unnatural when I try to read it.

Reply:
Since we had `getSelectedThumbnailPageNumbers` I was thinking of keeping the same naming but I can change it. Shorter name are better